examples/api/lib/material/dropdown_menu/dropdown_menu_entry_label_widget.0.dart

```diff
@@ -0,0 +1,91 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+/// Flutter code sample for the [DropdownMenuEntry] `labelWidget` property.
+
+enum ColorItem {
+  blue('Blue', Colors.blue),
+  pink('Pink', Colors.pink),
+  green('Green', Colors.green),
+  yellow('Yellow', Colors.yellow),
+  grey('Grey', Colors.grey);
+
+  const ColorItem(this.label, this.color);
+  final String label;
+  final Color color;
+}
+
+class DropdownMenuEntryLabelWidgetExample extends StatefulWidget {
+  const DropdownMenuEntryLabelWidgetExample({ super.key });
+
+  @override
+  State<DropdownMenuEntryLabelWidgetExample> createState() => _DropdownMenuEntryLabelWidgetExampleState();
+}
+
+class _DropdownMenuEntryLabelWidgetExampleState extends State<DropdownMenuEntryLabelWidgetExample> {
+  late final TextEditingController controller;
+
+  @override
+  void initState() {
+    super.initState();
+    controller = TextEditingController();
+  }
+
+  @override
+  void dispose() {
+    controller.dispose();
+    super.dispose();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    // Created by Google Bard from 'create a lyrical phrase of about 25 words that begins with "is a color"'.
+    const String longText = 'is a color that sings of hope, A hue that shines like gold. It is the color of dreams, A shade that never grows old.';
```

Comment:
If you made this a top level constant, you could access it from the test instead of duplicating it.

Reply:
Done.